src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -212,18 +212,55 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
       }
     }
 
+    "Passthrough tests on the GET /workspaces/%s/%s path" - {
+      s"OK status is returned for HTTP GET (dbGap workspace)" in {
+        val dao = new MockRawlsDAO
+        val rwr = dao.protectedRawlsWorkspaceResponse
```

Comment:
I'd rather these be right here in `WorkspaceApiServiceSpec`, especially because these tests that are using objects from `MockRawlsDAO` are not actually using `MockRawlsDAO`.